<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DetailWindow</class>
 <widget class="QWidget" name="DetailWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>448</width>
    <height>326</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Task Detail</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Task Details</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="info_layout">
     <item>
      <layout class="QHBoxLayout" name="title_layout">
       <item>
        <widget class="QLabel" name="title_titel">
         <property name="text">
          <string>Titel:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="title_label">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Task Title</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="title_spacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTextEdit" name="title_entry">
         <property name="minimumSize">
          <size>
           <width>175</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>100</width>
           <height>26</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="title_apply_button">
         <property name="text">
          <string>Change</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="status_layout">
       <item>
        <widget class="QLabel" name="status_title">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="status_label">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Pending</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="status_spacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QComboBox" name="status_menu">
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>Pending</string>
         </property>
         <item>
          <property name="text">
           <string>Pending</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Finished</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>In Progress</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="due_date_layout">
       <item>
        <widget class="QLabel" name="date_label">
         <property name="text">
          <string>Due Date:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="date_edit">
         <property name="displayFormat">
          <string>dd.MM.yyyy</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="description_layout">
       <item>
        <widget class="QLabel" name="description_label">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="description_spacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QTextEdit" name="description_entry">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>200</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="button_layout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="back_button">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save_button">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
